Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN --mount=type=cache,sharing=private,target=/var/lib/apt/lists \
   --mount=type=cache,sharing=private,target=/var/cache/apt \
   apt-get update && \
   apt-get install --no-install-recommends -y \
-  libjemalloc2 postgresql-client
+  curl libjemalloc2 postgresql-client
```

Comment:
It's kind of sad to be adding `curl` to the image just for the sake of the health check, but it's not the end of the world.